### PR TITLE
Updates for Orders CLI docs

### PR DESCRIPTION
**Pull request by cholmes**
**Related Issue(s):**

Closes #

**Proposed Changes:**

For inclusion in changelog (if applicable):

1.

Not intended for changelog:

1. Added section on STAC
2. added section on basemaps orders
3. changed to single quotes, and including `-`
4. various minor updates

**Diff of User Interface**

Old behavior:

New behavior:

**PR Checklist:**

- [x] This PR is as small and focused as possible
- [x] If this PR includes proposed changes for inclusion in the changelog, the title of this PR summarizes those changes and is ready for inclusion in the Changelog.
- [] I have updated docstrings for function changes and docs in the 'docs' folder for user interface / behavior changes
- [x] This PR does not break any examples or I have updated them

**(Optional) @mentions for Notifications:**

### CODE REVIEW

File: docs/cli/cli-orders.md [hunk 1]
```diff
@@ -530,19 +530,33 @@ The harmonize tool allows you to compare data to different generations of satell
...
+planet orders request psscene analytic_sr_udm2 --name 'Harmonized data' --id 20200925_161029_69_2223 --tools tools-harmonize.json
 ```
 
+## More options
+
 ### STAC Metadata
 
-TODO
+A relatively recent addition to Planet's orders delivery is the inclusion of [SpatioTemporal Asset Catalog](https://stacspec.org/en)
+(STAC) metadata in Orders. This provides a more standard set of JSON fields, that work with a number of
```
Comment (JuLeeAtPlanet) [inline docs/cli/cli-orders.md:544]:
```suggestion
(STAC) metadata in Orders. STAC metadata provides a more standard set of JSON fields that work with 
```

File: docs/cli/cli-orders.md [hunk 2]
```diff
@@ -530,19 +530,33 @@ The harmonize tool allows you to compare data to different generations of satell
...
 ```
 
+## More options
+
 ### STAC Metadata
 
-TODO
+A relatively recent addition to Planet's orders delivery is the inclusion of [SpatioTemporal Asset Catalog](https://stacspec.org/en)
+(STAC) metadata in Orders. This provides a more standard set of JSON fields, that work with a number of 
+[different tools](https://stacindex.org/ecosystem). The CLI `orders request` command currently requests
```
Comment (JuLeeAtPlanet) [inline docs/cli/cli-orders.md:545]:
```suggestion
many GIS and geospatial [STAC-enabled tools](https://stacindex.org/ecosystem). The CLI `orders request` command currently requests
```

File: docs/cli/cli-orders.md [hunk 3]
```diff
@@ -530,19 +530,33 @@ The harmonize tool allows you to compare data to different generations of satell
...
+STAC metadata by default, as the STAC files are small and often more useful than the default JSON metadata.
+You can easily turn off STAC output request with the `--no-stac` command:
+
+```
+planet orders request PSScene visual --name 'No STAC' --no-stac --id 20220605_124027_64_242b 
+```
+
+Currently this needs to be done for any 'composite' operation, as STAC output from composites is not yet
```
Comment (JuLeeAtPlanet) [inline docs/cli/cli-orders.md:553]:
```suggestion
For this current pre-release, STAC output from composites is not yet 
```

File: docs/cli/cli-orders.md [hunk 4]
```diff
@@ -530,19 +530,33 @@ The harmonize tool allows you to compare data to different generations of satell
...
+[different tools](https://stacindex.org/ecosystem). The CLI `orders request` command currently requests
+STAC metadata by default, as the STAC files are small and often more useful than the default JSON metadata.
+You can easily turn off STAC output request with the `--no-stac` command:
+
+```
+planet orders request PSScene visual --name 'No STAC' --no-stac --id 20220605_124027_64_242b 
+```
+
+Currently this needs to be done for any 'composite' operation, as STAC output from composites is not yet 
+supported (but is coming). You can explicitly add `--stac`, but it is the default, so does not need to
```
Comment (JuLeeAtPlanet) [inline docs/cli/cli-orders.md:554]:
```suggestion
supported. So turn it off STAC for any 'composite' operation.

You can explicitly add `--stac`, but it is the default, so does not need to
```

File: docs/cli/cli-orders.md [hunk 5]
```diff
@@ -530,19 +530,33 @@ The harmonize tool allows you to compare data to different generations of satell
...
+be included. For more information about Planet's STAC output see the [orders API documentation](https://developers.planet.com/apis/orders/delivery/#stac-metadata).
```
Comment (JuLeeAtPlanet) [inline docs/cli/cli-orders.md:555]:
```suggestion
be included. For more information about Planet's STAC output see the [Orders API documentation](https://developers.planet.com/apis/orders/delivery/#stac-metadata).
```

File: docs/cli/cli-orders.md [hunk 6]
```diff
@@ -584,6 +593,73 @@ API will just ignore them if they are included in a request.
 There is planned functionality to make it easy to 'update' an existing order, so you can easily grab a past order
 and use the CLI to customize it.
 
+### Basemaps Orders
+
+One of the newer features in Planet's orders API is the ability to [order basemaps](https://developers.planet.com/apis/orders/basemaps/).
+The CLI does not yet support a 'convenience' method to easily create the JSON - you unfortunately
+can't yet use `planet orders request` to help form an orders request. But all the other CLI functionality
+supports it, due to the modular design of the CLI.
```
Comment (JuLeeAtPlanet) [inline docs/cli/cli-orders.md:601]:
```suggestion
supports ordering basemaps through the Orders API.
```

File: docs/cli/cli-orders.md [hunk 7]
```diff
@@ -584,6 +593,73 @@ API will just ignore them if they are included in a request.
 There is planned functionality to make it easy to 'update' an existing order, so you can easily grab a past order
 and use the CLI to customize it.
 
+### Basemaps Orders
+
+One of the newer features in Planet's orders API is the ability to [order basemaps](https://developers.planet.com/apis/orders/basemaps/).
```
Comment (JuLeeAtPlanet) [inline docs/cli/cli-orders.md:598]:
```suggestion
One of the newer features in Planet's Orders API is the ability to [order basemaps](https://developers.planet.com/apis/orders/basemaps/).
```

File: docs/cli/cli-orders.md [hunk 8]
```diff
@@ -584,6 +593,73 @@ API will just ignore them if they are included in a request.
...
+      }
+  ],
+    "tools": [
+      {"merge": {}},
+      {"clip": {}}
+  ]
+}
+```
+
+Once you've got the JSON the other commands are all the same. Use create to submit it to the API:
```
Comment (JuLeeAtPlanet) [inline docs/cli/cli-orders.md:634]:
```suggestion
Once you've got the JSON, the other commands are all the same. Use create to submit it to the API:
```

**Review by JuLeeAtPlanet — APPROVED** [review]
Minor suggestions.

File: docs/cli/cli-orders.md (hunk 3 above)
Comment (cholmes) [reply docs/cli/cli-orders.md:553]: Rejecting this one, as this is an issue with Planet's API's, so won't be solved when SDK is out of pre-release.

**Pull request merged**